nginx.conf

```diff
@@ -0,0 +1,21 @@
+events {
```

Comment:
i would just leave this out entirely.

Reply:
The empty block looks funny to me as well, but NGINX results in an error when I don't have it:

```
nginx: [emerg] no "events" section in configuration
```